.github/workflows/Rebuild Database Templates.yml

```diff
@@ -88,8 +93,144 @@ jobs:
+      continue-on-error: true
+
+    - name: Read each repo last commit for repo Ed-Fi-ODS-Implementation and stored it in repositories.json 
+      working-directory: ./Ed-Fi-ODS-Implementation/
+      shell: pwsh
+      run: |
+
+        $oneeightyDaysAgo = (Get-Date).AddDays(-180).ToUniversalTime().ToString("yyyy-MM-ddTHH:mm:ssZ")        
+        Write-Host "oneeightyDaysAgo Date and Time in ISO 8601 format: $oneeightyDaysAgo"
+        $branchName = "${{ env.current_branch }}"
+        $sinceDate = $oneeightyDaysAgo 
+        $implementationUrl = "https://api.github.com/repos/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/commits?sha=$branchName&since=$sinceDate&page=1&per_page=200"
+        .\build.githubactions.ps1 'AddOrUpdateRepositoriesJson' -Url $implementationUrl -RepoName 'Ed-Fi-ODS-Implementation'
+
+    - name: Read each repo last commit for repo Ed-Fi-ODS and stored it in repositories.json 
+      working-directory: ./Ed-Fi-ODS-Implementation/
+      shell: pwsh
+      run: |
+
+        $oneeightyDaysAgo = (Get-Date).AddDays(-180).ToUniversalTime().ToString("yyyy-MM-ddTHH:mm:ssZ")
+        Write-Host "oneeightyDaysAgo Date and Time in ISO 8601 format: $oneeightyDaysAgo"
+        $branchName = "${{ env.current_branch }}"
+        $sinceDate = $oneeightyDaysAgo 
+        $odsUrl = "https://api.github.com/repos/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/commits?sha=$branchName&since=$sinceDate&page=1&per_page=200"
+        .\build.githubactions.ps1  'AddOrUpdateRepositoriesJson' -Url $odsUrl -RepoName 'Ed-Fi-ODS'
+
+    - name: Read each repo last commit for repo Ed-Fi-Extensions and stored it in repositories.json 
+      working-directory: ./Ed-Fi-ODS-Implementation/
+      shell: pwsh
+      run: |
+
+        $oneeightyDaysAgo = (Get-Date).AddDays(-180).ToUniversalTime().ToString("yyyy-MM-ddTHH:mm:ssZ")
+        Write-Host "oneeightyDaysAgo Date and Time in ISO 8601 format: $oneeightyDaysAgo"
+        $branchName = "${{ env.current_branch }}"
+        $sinceDate = $oneeightyDaysAgo        
+        $extensionsUrl = "https://api.github.com/repos/Ed-Fi-Alliance-OSS/Ed-Fi-Extensions/commits?sha=$branchName&since=$sinceDate&page=1&per_page=200"
+        .\build.githubactions.ps1  'AddOrUpdateRepositoriesJson' -Url $extensionsUrl -RepoName 'Ed-Fi-Extensions'
+
+    - name: Check for IscommitChanged in repositories.json
+      working-directory: ./Ed-Fi-ODS-Implementation/
+      shell: pwsh
+      run: |
+        $FilePath = "repositories.json"
+        
+        # Read and parse the JSON file
+        $jsonContent = Get-Content -Path $FilePath | ConvertFrom-Json
+        
+        # Check if any repository has IscommitChanged set to true
+        $isCommitChanged = $jsonContent.repositories | Where-Object { $_.IscommitChanged -eq 'true' }
+
+        if ($isCommitChanged) {
+          # Set environment variable if any repository has IscommitChanged set to true
+          echo "IsSameCommitinAllthreeRepository=false" >> $Env:GITHUB_ENV
```

Comment:
can you change IsSameCommitinAllthreeRepository to NoNewCommitsInAllThreeRepositories for better readability.